src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveCompletionItemProvider.cs

```diff
@@ -27,6 +27,25 @@ internal class DirectiveCompletionItemProvider : RazorCompletionItemProvider
             CSharpCodeParser.TagHelperPrefixDirectiveDescriptor,
         };
 
+        // internal for testing
+        // Do not forget to update both insert and display text !important
+        internal static readonly IReadOnlyDictionary<string, (string InsertText, string DisplayText)> s_singleLineDirectiveSnippets = new Dictionary<string, (string InsertText, string DisplayText)>(StringComparer.Ordinal)
```

Comment:
Storing snippets as such tuples is a bit hacky IMO, but I didn't come up with any better variant. They are hardcoded anyway, but if you have better approach here, it will be welcome

Reply:
You could use a record, but honestly the code would look almost identical so this is fine to me.